certificate-renewal/Certificate Renewal JSON contract.md

```diff
@@ -0,0 +1,195 @@
+# CD Renew JSON contract
+
+## Initial Request (PRELOADDATA):
+```json
+{
+  "rgState": "PRELOADDATA",
+  "powerOnFileName": "BANNO.CD.RENEW.V1.POW",
+  "userChrList":[
+    {"id": 1, "value": "0123456789S0123"}
+  ],
+  "userNumList": [],
+  "rgSession": 1
+}
+```
+ - userCharList[1]: 10-digit account number, 'S' and 4-digit share ID
+	 - Represents the target certificate or club share
+
+
+## Successful Response:
+```json
+{
+  "results": {
+    "currentState": {
+      "name": "My Wonderful CD",
+      "maturityDate": "01/01/21",
```

Comment:
`"term": "24 months",`  can some version of this be added to the currentState again?

Reply:
@reuelcj como este?

```
    "currentState": {
      "name": "My Wonderful CD",
      "maturityDate": "01/01/21",
      "minimumBalance": "12345.67",
      "currentStatus": "Renew",
      "currentTerm": "24 months",
      "transferSLId": "1234567890S0010",
      "transferAmount": "12345.67"
    },
```

Reply:
perfect